### PR TITLE
Adding instruction about running the app on simulator from SpatialOS menu

### PR DESCRIPTION
**Pull request by gkassabli**
#### Description
Documentation describing launching on iOS Device from menu - https://github.com/spatialos/gdk-for-unity/pull/670

#### Tests
Ensure documentation looks as expected

### CODE REVIEW

File: docs/content/mobile/ios/local-deploy.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     * If you choose **Build**, instead of **Build and Run**, Unity generates a XCode project and opens the folder containing the project.
   1. Once the game is deployed and started on the Simulator, you see an empty text field and a **Connect** button: Select **Connect**.
 
-    > You don’t need to enter anything in the text field.
+    > You don’t need to enter anything in the text field,
+
+  1. Once you've installed or ran app on the Simulator you can run **SpatialOS** > **Launch Mobile Client** > **iOS Client** to launch mobile client with IP address field prefilled with a value from **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**.
```
Comment (jessicafalk) [inline docs/content/mobile/ios/local-deploy.md]: this seems to be the wrong place. I would maybe add it as a call-out underneath the last point:

something along:
```
> **Note**:  We provide a setting in the GDK Tools configuration window to fill in your local IP address. In the Unity Editor, go to  **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**. Once you have built the app via XCode, you can launch the iOS simulator via **SpatialOS** > **Launch Mobile Client** > **iOS Client** in the Unity Editor. This starts your app on your iOS simulator and passes in the IP address that you defined as an argument. 
```

probably best for @ElleEss  to do a check through that text as well

File: docs/content/mobile/ios/local-deploy.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     > You don’t need to enter anything in the text field.
 
   1. Play the game on the Simulator.
+  1. Once you've installed or ran app on the Simulator you can run **SpatialOS** > **Launch Mobile Client** > **iOS Client** to launch mobile client with IP address field prefilled with a value from **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**.
```
Comment (ElleEss) [inline docs/content/mobile/ios/local-deploy.md]:
```suggestion
  1. Once you've installed or run an app on the Simulator, back in your Unity Editor, you can run **SpatialOS** > **Launch Mobile Client** > **iOS Client** to launch a mobile client with the IP address field prefilled with a value from **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**.
```

File: docs/content/mobile/ios/local-deploy.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     > You don’t need to enter anything in the text field.
 
   1. Play the game on the Simulator.
```
Comment (ElleEss) [inline docs/content/mobile/ios/local-deploy.md]: These numbers are rendering oddly in GitHub - are they OK in improbadoc locally?

Comment (gkassabli) [reply docs/content/mobile/ios/local-deploy.md]: Yeah, it looked fine when I've tested it locally. I'll double-check

File: docs/content/mobile/ios/local-deploy.md [hunk 4]
```diff
@@ -42,6 +42,8 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
...
+  1. Once you've installed or run an app on the Simulator, back in your Unity Editor, you can run **SpatialOS** > **Launch Mobile Client** > **iOS Client** to launch a mobile client with the IP address field prefilled with a value from **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**.
```
Comment (jessicafalk) [inline docs/content/mobile/ios/local-deploy.md]: make the same changes here as yo udid for the pr

Comment (ElleEss) [inline docs/content/mobile/ios/local-deploy.md]: Do you mean add the text like in this PR?  https://github.com/spatialos/gdk-for-unity/pull/678/files/

**Pull request closed without merging**